### PR TITLE
Ignored files in meta.yaml

### PR DESCRIPTION
**Pull request by frenzymadness**
* Use `meta.yaml` to store a list of ignored files in the pyvo-data
* Split `metadata` from `data` and load `metadata` in separated function

### CODE REVIEW

File: pyvodb/load.py [hunk 1]
```diff
@@ -53,21 +53,30 @@ def dict_from_directory(directory, root, ignored_files=()):
     return data
 
 
+def load_meta_file(directory):
+        metafile = os.path.join(directory, 'meta.yaml')
+        with open(metafile) as f:
+            info = yaml.load(f, Loader=YAML_SAFE_LOADER)
+        info['ignored_files'].append('meta.yaml')
```
Comment (encukou) [inline pyvodb/load.py]: This removes `meta` from the result of `load_from_directory`. I'd rather have it available.

File: pyvodb/load.py [hunk 2]
```diff
@@ -53,21 +53,30 @@ def dict_from_directory(directory, root, ignored_files=()):
...
+        return info
+
+
 def load_from_directory(db, directory):
+    metadata = load_meta_file(directory)
     data = dict_from_directory(
-        '.', directory, ignored_files=['.git', 'README', 'tests'])
-    load_from_dict(db, data)
+        '.', directory, ignored_files=metadata['ignored_files'])
```
Comment (encukou) [inline pyvodb/load.py]: Can this be `ignored_files=metadata.get('ignored_files', ['.git', 'README', 'tests'])`, for backwards compatibility?

Comment (frenzymadness) [inline pyvodb/load.py]: Sure!

File: pyvodb/load.py (hunk 1 above)
Comment (frenzymadness) [inline pyvodb/load.py]: Ok, I'll remove this line, so `meta.yaml` will be also loaded to `data`.

**Comment by frenzymadness** [comment]
See mentioned changes in the fixup commit.

**Comment by encukou** [comment]
One more fixup, does it look OK to you?

**Comment by frenzymadness** [comment]
Yes! Let's merge it.

**Pull request merged**